### ISSUE TITLE
paper discussion

### ISSUE THREAD

**Issue by JadTawil-theonly** [hungpham2511/toppra-path-following#1]
Hello Hung,
Are you open to discussing the paper sometime? :)

**Comment by hungpham2511** [comment]
Certainly, feel free to post any issues/clarifications you are having. I will be happy to answer.

Bests,

Hung

________________________________
